x-pack/plugins/cases/common/api/cases/user_actions/response.ts

```diff
@@ -92,15 +92,22 @@ export type CaseUserActionAttributesWithoutConnectorId = rt.TypeOf<
   typeof CaseUserActionBasicWithoutConnectorIdRt
 >;
 export type CaseUserActionStatsResponse = rt.TypeOf<typeof CaseUserActionStatsRt>;
-export type CaseUserActionsResponse = rt.TypeOf<typeof CaseUserActionsResponseRt>;
-export type CaseUserActionResponse = rt.TypeOf<typeof CaseUserActionResponseRt>;
+export type UserActions = rt.TypeOf<typeof UserActionsRt>;
+export type UserAction = rt.TypeOf<typeof UserActionRt>;
 export type CaseUserActionsDeprecatedResponse = rt.TypeOf<
   typeof CaseUserActionsDeprecatedResponseRt
 >;
 export type CaseUserActionDeprecatedResponse = rt.TypeOf<typeof CaseUserActionDeprecatedResponseRt>;
-export type CaseUserActionInjectedAttributes = rt.TypeOf<typeof CaseUserActionInjectedAttributesRt>;
+export type UserActionAttributesWithInjectedId = rt.TypeOf<
```

Comment:
Maybe we should rename it to `UserActionAttributes` as we did with `CaseAttributes`.